packages/ui/src/views/user-management/components/dialogs/components/create-user/create-user.tsx

```diff
@@ -0,0 +1,105 @@
+import { SubmitHandler, useForm } from 'react-hook-form'
+
+import { Button, ButtonGroup, ControlGroup, Dialog, Fieldset, FormWrapper, Input } from '@/components'
+import { useStates } from '@/views/user-management/providers/state-provider'
+import { useUserManagementStore } from '@/views/user-management/providers/store-provider'
+import { zodResolver } from '@hookform/resolvers/zod'
+import {
+  createNewUserSchema,
+  NewUserFields
+} from '@views/user-management/components/dialogs/components/create-user/schema'
+
+interface CreateUserDialogProps {
+  handleCreateUser: (data: NewUserFields) => void
+  open: boolean
+  onClose: () => void
+}
+
+export function CreateUserDialog({ handleCreateUser, open, onClose }: CreateUserDialogProps) {
+  const { useTranslationStore } = useUserManagementStore()
+  const { t } = useTranslationStore()
+
+  const { loadingStates, errorStates } = useStates()
+  const { isCreatingUser } = loadingStates
+  const { createUserError } = errorStates
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm<NewUserFields>({
+    resolver: zodResolver(createNewUserSchema(t)),
+    mode: 'onChange',
+    defaultValues: { uid: '', email: '', display_name: '' }
+  })
+
+  const onSubmit: SubmitHandler<NewUserFields> = data => {
+    handleCreateUser(data)
+  }
+
+  return (
+    <Dialog.Root open={open} onOpenChange={onClose}>
+      <Dialog.Content className="max-w-xl">
+        <Dialog.Header>
+          <Dialog.Title className="font-medium">
+            {t('views:userManagement.createUser.title', 'Add a new user')}
+          </Dialog.Title>
+        </Dialog.Header>
+
+        <FormWrapper onSubmit={handleSubmit(onSubmit)}>
+          <Fieldset>
+            <ControlGroup>
```

Comment:
```suggestion
```
The `Input` component already contains a `ContentGroup` by default, so we don't need to add it manually 
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/16259b49-00ff-421d-9ce9-95a3127a1d8b" />